src/PowerPlots.jl

```diff
@@ -26,9 +26,27 @@ _IM = InfrastructureModels
 const nx = PyCall.PyNULL()
 const scipy = PyCall.PyNULL()
 
+# Create our module level logger (this will get precompiled)
+const _LOGGER = Memento.getlogger(@__MODULE__)
+
+"Suppresses information and warning messages output by PowerPlots, for fine grained control use the Memento package"
+function silence()
```

Comment:
Is it possible to exclude `silence` and `logger_config!` from exports so that you must call with `PowerPlots.silence()` etc first? I'm not too familiar with how `export` works if a user does `using PowerPlots`...does the export qualify names when `using` so that they don't need the module name before it?

Reply:
Committed changes that made `export.jl` more cleaner and hid the logger functions to avoid conflict (e.g. users now have to qualify with `PowerPlots.silence()`)

Reply:
@noahrhodes see 4a4f70e5e33b5b317fa46f278f2967e1602b34eb and let me know what you think

Reply:
Also, I realized there was a typo in the previous commit 9504fd7db27cf8d2f2ddc2e725e2475107855fe7 that I fixed

Reply:
I think that's a good way to do it.  Maybe move the  code related to exports into `export.jl`? So the function definition `_hide_function` and all of the function calls are in the same place.  This makes it easy to check which functions are not being exported.